wiki/Disambiguation/Mod/en.md

```diff
@@ -2,6 +2,6 @@
 
 **Mod** can have multiple meanings:
 
-- A moderator, such as an [NAT](/wiki/People/The_Team/Nomination_Assessment_Team) or [GMT](/wiki/People/The_Team/Global_Moderation_Team) member.
-- [The feedback](/wiki/Modding) given by a person who looks over a beatmap.
-- A [game modifier](/wiki/Game_modifier).
+- A moderator, such as a [NAT](/wiki/People/The_Team/Nomination_Assessment_Team) or [GMT](/wiki/People/The_Team/Global_Moderation_Team) member
```

Comment:
dont see why this change was neccessary? English uses a vs an based off the pronounciation of the acronym itself and not the words in the acronym afaik.

Reply:
I know, it's... it's just that I've thought for a second that it reads as `'næt`. duh

Reply:
huh, i was pronouncing it "N-A-T". so i actually dont know which one is correct here. either way it doesnt matter much